tests/common/storages/utils.py

```diff
@@ -49,6 +68,7 @@ def assert_sample_files(
         with file_dict.open() as f:
             assert content == f.read()
         # read via various readers
+        print(item)
```

Comment:
@sh-rp @sultaniman please do not let debug prints into the `devel`. if this goes into release we'll need a quick post release to fix it. and this is PITA :)